src/main/java/wolox/training/TrainingApplication.java

```diff
@@ -0,0 +1,13 @@
+package wolox.training;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class TrainingApplication {
+
+	public static void main(String[] args) {
```

Comment:
As a good practice we use 4 spaces instead of a tab character, you can set this up in IntelliJ like this: 

![image](https://user-images.githubusercontent.com/43755493/74337987-1dd6f200-4d80-11ea-8493-6d6a4978a72d.png)

Shortcut to this screen: Shift + Shift (search menu) + Code Style > Code Style (Preferences) 😄

Reply:
:+1: